src/adaptation/icap/Client.cc

```diff
@@ -12,7 +12,7 @@
 
 void Adaptation::Icap::InitModule()
 {
-    debugs(93,2, HERE << "module enabled.");
+    debugs(93,2, "module enabled.");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.